modules/acquia_cms_site_studio/acquia_cms_site_studio.install

```diff
@@ -17,8 +17,11 @@ use Drupal\user\Entity\Role;
  * @throws Exception
  */
 function acquia_cms_site_studio_install() {
-  $site_studio_api_key = getenv('SITESTUDIO_API_KEY');
-  $site_studio_org_key = getenv('SITESTUDIO_ORG_KEY');
+  // If site studio key available in the environment variables
+  // then pick it from there else check for settings overrides.
+  $config = \Drupal::config('cohesion.settings');
+  $site_studio_api_key = getenv('SITESTUDIO_API_KEY') ?? $config->get('api_key');
```

Comment:
@chandan-singh7929 Isin't think this would give error if acquia_cms_site_studio module is installed without adding values to configuration ?

Reply:
No, it will not since we are allowing site studio to be install without SS key as well. User has ability to set the same later on using UI